src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -26,6 +26,7 @@ module Log =
         | WriteConflict of Measurement
         | Slice of Measurement
         | Batch of slices : int * Measurement
+        | Last of Measurement
```

Comment:
try to align ordering with match statements in general (and prob reads before writes?)